### ISSUE TITLE
publish asciinema screencast of getting started

### ISSUE THREAD

**Issue by andychu** [oilshell/oil#789]
`build/dev.sh minimal`

Should take two seconds ?

Maybe this will help people get started.

**Comment by andychu** [comment]
inspired by #623

**Comment by jyn514** [comment]
> And it will also force us to clean up the spew.

+1 to this, a good start would be getting rid of the compile warnings I think.

**Comment by andychu** [comment]
OK I just got rid of many compile warnings, but not all.  Still more to do

A lot of it was because of "hollowing out the python interpreter" -- Python method tables no longer referenced C functions, leading to unused function warnings

**Comment by jyn514** [comment]
I actually meant fixing the warnings by removing the functions 😆 but C compiles quickly enough that this works too.

**Comment by andychu** [comment]
Yeah it's a fork for us only, so I want to minimize the diff between `native/posixmodule.c` and the `Python-2.7.13/` version... It helps in auditing what we changed!

**Comment by andychu** [comment]
Published them here, let me know what you think!

https://oilshell.zulipchat.com/#narrow/stream/121539-oil-dev/topic/oil-dev.20screencasts

https://asciinema.org/a/zoWDffn5iK0SdFX2605Vzc8Wn

https://asciinema.org/a/UuY8BS6Dj6BSx3wG5eHTrwsHO
